### ISSUE TITLE
FEATURE REQUEST: Refresh button in the header/context menu

### ISSUE THREAD

**Issue by degreeze** [Ross-Patterson/Portals-Desktop-Organization#365]
### Check the Help Site and GitHub Issues/Discussions

- [X] I have checked and found nothing.

### What is the enhancement/feature you would like to see?

In some cases the system file events don't work properly and files do not appear in the portal as expected. The Reload is a reliable way to deal with this but for now its button is only available in the hamburger menu.
It would be nice to have an option to place this button in the header like Search, Customize and etc.

**Comment by ghost** [comment]
If I'm correct, you should be able to press F5 for the same function

**Comment by degreeze** [comment]
> If I'm correct, you should be able to press F5 for the same function

Indeed! I've tried it, but maybe I failed to properly focus on the portal when I did it 🤔

Still, sometimes quick access with GUI is preferable, it's more "tactile". Like Ctrl+S is a hotkey for search but it has a separate quick button for it.

**Comment by Ross-Patterson** [comment]
This can be easily accommodated - I will add it to the backlog.
